src/pages/register/Register.css

```diff
@@ -0,0 +1,32 @@
+.registerPage {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 80vh; /* Set the height of the container to 80% of the viewport height */
+  }
+  
+  .registerContainer {
+    width: 300px; /* Adjust the width of the form container as needed */
+  }
+  
+  .inputField {
+    width: 100%;
+    margin-bottom: 10px;
+    padding: 8px;
+    box-sizing: border-box;
+  }
+  
+  .submitButton {
+    width: 100%;
+    padding: 10px;
+    background-color: #3498db;
+    color: #fff;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+  
+  .submitButton:hover {
+    background-color: #2980b9;
```

Comment:
styling looks good, eventually we'll refer to colors in the UI kit but this works for now!

Reply:
Thanks!